tfhe/src/typed_api/mod.rs

```diff
@@ -0,0 +1,51 @@
+#![allow(unused_doc_comments)]
+#![cfg_attr(fmt, rustfmt::skip)]
+#![cfg_attr(doc, feature(doc_auto_cfg))]
+#![cfg_attr(doc, feature(doc_cfg))]
+#![allow(clippy::bool_assert_comparison)]
+#![allow(clippy::assign_op_pattern)]
+#![allow(clippy::eq_op)]
+#![allow(clippy::assign_op_pattern)]
```

Comment:
I'm not a fan of global disables for lints, any way we can make this more local ?

Reply:
moved into the file they were needed [typed_api/booleans/tests.rs] (and removed the ones that were not needed anymore)